modules/nodes-vsphere-machine-set-concept-static-ip.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/node-tasks.adoc
+
+:_content-type: CONCEPT
+[id="nodes-vsphere-machine-set-concept-static-ip_{context}"]
+= How a machine set scales machines with configured static IP addresses
```

Comment:
```suggestion
= Scaling machines with configured static IP addresses
```

Reply:
Thanks. I'll go with your title here to avoid the use of "How".